src/components/RegisterForm.component/SellerRegistration.js

```diff
@@ -1,124 +1,151 @@
+                        </div>
+                    </div>
+
+                    <div className="form-inputs">
+                        <label>Password</label>
+                        <input
+                            type="password"
+                            name="password"
+                            value={userDetails.password}
+                            onChange={handleChange}
+                            placeholder="Enter Password"
+                            onKeyUp={handleChange}
+                            autoComplete="off"
+                            onBlur={handleValidations}
+                        />{' '}
+                        <div style={{height: '36px'}}>
+                            {errors.password && <p>{errors.password}</p>}
+                        </div>
+                    </div>
+
+                    <div className="form-inputs">
+                        <label>Confirm Password</label>
+                        <input
+                            type="password"
+                            name="confirmPassword"
+                            value={userDetails.confirmPassword}
+                            onChange={handleChange}
+                            placeholder="Confirm Password"
+                            onKeyUp={handleChange}
+                            autoComplete="off"
+                            onBlur={handleValidations}
+                        />{' '}
+                        <div style={{height: '36px'}}>
+                            {errors.confirmPassword && <p>{errors.confirmPassword}</p>}
+                        </div>
+                    </div>
 
-        <div className="form-inputs">
-          <label>Phone Number</label>
-          <input
-            type="number"
-            placeholder="Phone Number"
-            {...register('phoneNumber', {
-              required: 'Phone Number is Required',
-              minLength: {
-                value: 10,
-                message: 'Phone Number must be 10 characters',
-              },
-              maxLength: {
-                value: 10,
-                message: 'Phone Number must be 10 characters',
-              },
-            })}
-            onKeyUp={() => {
-              trigger('phoneNumber')
-            }}
-          />{' '}
-          <div style={{ height: '36px' }}>
-            {errors.phoneNumber && <p>{errors.phoneNumber.message}</p>}
-          </div>
+                    <div className="form-inputs">
+                        <label>Phone Number</label>
+                        <input
+                            type="number"
+                            name="phoneNumber"
+                            value={userDetails.phoneNumber}
+                            onChange={handleChange}
+                            placeholder="Phone Number"
+                            onKeyUp={handleChange}
+                            autoComplete="off"
+                            onBlur={handleValidations}
```

Comment:
no need for onKeyup property, the exact behaviour what we required from validation is not working properly